libs/shared/ui/src/lib/styles/base/icons.scss

```diff
@@ -475,3 +475,7 @@
 .icon-solid-lock {
   @include fa-icon-solid($fa-var-lock);
 }
+
+.icon-solid-wand-magic-sparkles:before {
```

Comment:
you can remove it

Reply:
https://github.com/Qovery/console/pull/1265/commits/712b384bbe93abfa50ca67187be1e344a5a6bdcc